src/LiveApi.js

```diff
@@ -13,6 +13,8 @@ let WebSocket = typeof window !== 'undefined' ? window.WebSocket : MockWebSocket
 const shouldIgnoreError = error =>
     error.message.includes('You are already subscribed to') ||
         error.message.includes('Input validation failed: forget');
+const shouldNotThrow = json => 
+    json.error.code === 'SelfExclusion' && json.msg_type === 'authorize';
 
```

Comment:
@borisyankov supposedly there should be an error message displayed telling users that the intended account has been excluded. Perhaps a flash like the one we have in self exclusion. 😉